src/data/repack.cc

```diff
@@ -15,20 +15,16 @@
  *
  */
 #include "data/repack.h"
+
 #include <stdlib.h>
 
+#include "util/math.h"
```

Comment:
I try to format the according to Google C++ Style Guide: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes
Which commit exactly has different order?